assets/css/v2/triptych/screen_container.scss

```diff
@@ -1,7 +1,16 @@
 .screen-container {
   position: relative;
-  width: 1080px;
+  width: 3240px;
   height: 1920px;
   margin: 0px auto;
   overflow: hidden;
 }
+
+.screen-container-blink {
```

Comment:
Oops, actually I didn't need to add this. We'll never use a "blink" element on this screen type. Will remove